### PR TITLE
Add NFR, Use cases, Manual testing

### PR DESCRIPTION
**Pull request by NigelYeoTW**

### CODE REVIEW

**Review by wongwh2002 — COMMENTED** [review]
lgtm here

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -651,14 +655,25 @@ Use case ends.
...
+
+Use case ends.
+## Non-Functional Requirements
 
 1. A user should be alerted of the correct command format whenever an invalid command is encountered.
 2. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be 
 able to accomplish most of the tasks faster using commands than using the mouse.
 3. The system should be able to run on Windows, macOS, and Linux
+4. The system should be able to be used easily by a new user
+5. The program should not have use of language deemed offensive
```
Comment (TPH777) [inline docs/DeveloperGuide.md:674]: Maybe rephrase to `The program should not use language deemed offensive.`?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -651,14 +655,25 @@ Use case ends.
...
+6. The program should be able to recover as much data as possible in the event of corruption
```
Comment (TPH777) [inline docs/DeveloperGuide.md:675]: Maybe specify that it is the corruption of data in storage?

**Review by TPH777 — COMMENTED** [review]
Just some minor suggestions, else lgtm!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (NigelYeoTW) [reply docs/DeveloperGuide.md]: fixed!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (NigelYeoTW) [reply docs/DeveloperGuide.md]: dones.
